orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -345,3 +344,131 @@ def test_gpu_reserved_memory_with_torch_no_grad():
+            self.fc1_2 = torch.nn.Linear(hidden_size, num_classes)
+
+            self.fc2_1 = torch.nn.Linear(input_size, hidden_size)
+            self.relu2 = torch.nn.ReLU()
+            self.fc2_2 = torch.nn.Linear(hidden_size, num_classes)
+
+            self.fc3_1 = torch.nn.Linear(input_size, hidden_size)
+            self.relu3 = torch.nn.ReLU()
+            self.fc3_2 = torch.nn.Linear(hidden_size, num_classes)
+
+        def forward(self, input1, input2, input3):
+            out1 = self.fc1_2(self.relu1(self.fc1_1(input1)))
+            out2 = self.fc2_2(self.relu2(self.fc2_1(input2)))
+            out3 = self.fc3_2(self.relu3(self.fc3_1(input2)))
+            return CustomClass(out1, out2, out3)
+
+    N, D_in, H, D_out = 64, 784, 500, 10
+    model = NeuralNetCustomClassOutput(D_in, H, D_out).to(device)
+    model = ORTModule(model)
+    x = torch.randn(N, D_in, device=device)
+    y = torch.randn(N, D_in, device=device)
+    z = torch.randn(N, D_in, device=device)
+
+    with pytest.raises(RuntimeError) as runtime_error:
+        model(x, y, z)
+    assert 'ORTModule does not support the following model output type' in str(runtime_error.value)
+
+def test_dynamic_axes_config():
+    device = 'cuda'
+
+    def assert_is_dynamic_axes(model):
+        # Check inputs
+        for inp in model._onnx_training.graph.input:
+            shape = inp.type.tensor_type.shape
+            if shape:
+                for dim in shape.dim:
+                    if dim.dim_param and not isinstance(dim.dim_param, str):
+                        return False
+
+        # Check outputs
+        for out in model._onnx_training.graph.output:
+            shape = out.type.tensor_type.shape
+            if shape:
+                for dim in shape.dim:
+                    if dim.dim_param and not isinstance(dim.dim_param, str):
+                        return False
+        return True
+
+    # Model 1
+    N, D_in, H, D_out = 64, 784, 500, 10
+    model = NeuralNetSinglePositionalArgument(D_in, H, D_out).to(device)
+    model = ORTModule(model)
+    x = torch.randn(N, D_in, device=device)
+    output = model(x)
+    assert output is not None
+    assert assert_is_dynamic_axes(model)
+    del model, output
+
+    # Model 2
+    config = AutoConfig.from_pretrained(
+            "bert-base-uncased",
+            num_labels=2,
+            num_hidden_layers=1,
+            output_attentions = False,
+            output_hidden_states = False,
+    )
+    model = BertForSequenceClassification.from_pretrained(
+        "bert-base-uncased",
+        config=config,
+    )
+    model = ORTModule(model).to(device)
+
+    x = torch.randint(0, 100, (32, 64), dtype=torch.long, device=device)
+    y = torch.randint(0, 100, (32, 64), dtype=torch.long, device=device)
+    z = torch.randint(0, 1, (32,), dtype=torch.long, device=device)
+    output = model(x, y, None, None, None, None, z)
+    assert output is not None
+    assert assert_is_dynamic_axes(model)
```

Comment:
May we rename the function to ```is_dynamic_axes``` instead or put the ```assert``` inside the function ```assert_is_dynamic_axes``` so that the name is representative of what it is doing?